File: python/payment.py
class Payment:
    def __init__(self, payment_id, application_id, amount, payment_date, payment_status):
        self.payment_id = payment_id
        self.application_id = application_id
        self.amount = amount
        self.payment_date = payment_date
        self.payment_status = payment_status

    def make_payment(self):
        print("Payment made.")

    def get_payment_status(self):
        print(f"Payment status: {self.payment_status}")

    def generate_receipt(self):
        print("Receipt generated.")

File: GoaOnline/src/Payment.java
public class Payment {
    private int paymentId;
    private int applicationId;
    private double amount;
    private String paymentDate;
    private String paymentStatus;

    // Constructor
    public Payment(int paymentId, int applicationId, double amount, String paymentDate, String paymentStatus) {
        this.paymentId = paymentId;
        this.applicationId = applicationId;
        this.amount = amount;
        this.paymentDate = paymentDate;
        this.paymentStatus = paymentStatus;
    }

    // Methods
    public void makePayment() {
        System.out.println("Payment made.");
    }

    public void getPaymentStatus() {
        System.out.println("Payment status: " + paymentStatus);
    }

    public void generateReceipt() {
        System.out.println("Receipt generated.");
    }
}
